dist/stratus.js

```diff
@@ -2150,48 +2168,48 @@ Stratus.Internals.OnScroll = _.once(function (elements) {
   // Execute the methods for every registered object ONLY when there is a
   // change to the viewPort
   Stratus.Environment.on('change:viewPortChange', function (event, model) {
-    if (model.get('viewPortChange')) {
-      model.set('lastScroll', Stratus.Internals.GetScrollDir())
-
-      // Cycle through all the registered objects an execute their function
-      // We must use the registered onScroll objects, because they get removed
-      // in some cases (e.g. lazy load)
-      // TODO: remove logic of RegisterGroup
-      if (!Stratus.Environment.get('production')) {
-        console.log('Remove RegisterGroup Logic:')
-      }
-      let elements = Stratus.RegisterGroup.get('OnScroll')
-
-      _.each(elements, function (obj) {
-        if (typeof obj !== 'undefined' && _.has(obj, 'method')) {
-          obj.method(obj)
-        }
-      })
-      model.set('viewPortChange', false)
-      model.set('windowTop', jQuery(Stratus.Environment.get('viewPort') || window).scrollTop())
+    if (!model.get('viewPortChange')) {
+      return
     }
-  })
+    model.set('lastScroll', Stratus.Internals.GetScrollDir())
 
-  // jQuery Binding
-  if (typeof jQuery === 'function' && jQuery.fn) {
-    jQuery(Stratus.Environment.get('viewPort') || window).scroll(function () {
-      /* *
-      if (!Stratus.Environment.get('production')) {
-        console.log('scrolling:', Stratus.Internals.GetScrollDir())
-      }
-      /* */
-      if (Stratus.Environment.get('viewPortChange') === false) {
-        Stratus.Environment.set('viewPortChange', true)
-      }
-    })
+    // Cycle through all the registered objects an execute their function
+    // We must use the registered onScroll objects, because they get removed
+    // in some cases (e.g. lazy load)
+    // TODO: remove logic of RegisterGroup
+    let elements = Stratus.RegisterGroup.get('OnScroll')
 
-    // Resizing can change what's on screen so we need to check the scrolling
-    jQuery(Stratus.Environment.get('viewPort') || window).resize(function () {
-      if (Stratus.Environment.get('viewPortChange') === false) {
-        Stratus.Environment.set('viewPortChange', true)
+    _.each(elements, function (obj) {
+      if (typeof obj !== 'undefined' && _.has(obj, 'method')) {
+        obj.method(obj)
       }
     })
-  }
+    model.set('viewPortChange', false)
+    model.set('windowTop', jQuery(Stratus.Environment.get('viewPort') || window).scrollTop())
```

Comment:
Will we be moving away from this jQuery reference eventually?
There are equivalents such as https://stackoverflow.com/questions/11193453/find-the-vertical-position-of-scrollbar-without-jquery#answer-11193613
or https://www.w3schools.com/jsref/prop_element_scrolltop.asp

Reply:
One day!

Reply:
To make the one day, one day come, I'll always comment on jQuery code I see :p